docs/labwc-actions.5.scd

```diff
@@ -147,6 +147,56 @@ Actions are used in menus and keyboard/mouse bindings.
 *<action name="None" />*
 	If used as the only action for a binding: clear an earlier defined binding.
 
+# CONDITIONAL ACTIONS
+
+Actions that execute other actions. Used in keyboard/mouse bindings.
+
+*<action name="If">*
+	This action will execute one set of actions if the focused window
+	matches the criteria, or another if it does not.
+
+	The arguments are as follows:
+
+	```
+	<action name="If">
+	  <query/>
+	  <then><action/></then>
+	  <else><action/></else>
+	</action>
+	```
+
+	*query*
+		Define a query with zero or more conditions. All conditions must
+		be evaluated as true in order for the window to match this
+		query. Multiple queries can be defined.
+
+		Pattern matching is done according to glob(7) and is
```

Comment:
Have we used glob(7) or g_pattern_match_simple()? 
On phone, so can't easily scroll down and check.
match_glob() uses the latter and I'm not sure that it's "glob(7)" compliant

Reply:
match_glob(), but I remember we talked about switch it from non-standard glib function to glob.

Reply:
Can't remember `fnmatch()` discussion, but happy to move that way.
We have to keep the pattern match find `foo.bar.baz` given `foo.*.baz` (so glob rather than regex). Other that, I don't think we have any backward compatibility considerations.

Did we consider `<title type="regex">` for `<action name="If">` at one point? Have slept since.
http://openbox.org/wiki/Help:Actions#If

Reply:
> Can't remember fnmatch() discussion, but happy to move that way.

During this PR?

> Did we consider <title type="regex"> for <action name="If"> at one point? Have slept since.

Not yet. IMO, basic pattern matching is fine for now.

Reply:
Happy to move to fnmatch as part of this PR or one of could change it later. I’m easy 😊

Reply:
Let's get over this, then. Updated.